testing/tests/DevExpress.ui.widgets.editors/textEditorParts/mask.tests.js

```diff
@@ -1153,6 +1153,33 @@ QUnit.module("value", moduleConfig, () => {
         assert.equal(valueChangedFired, 1, "change fired once on pressing enter key");
     });
 
+    QUnit.test("value option change and input value change by typing should trigger onValueChange (T833415)", (assert) => {
```

Comment:
```suggestion
    QUnit.test("value option change and input value change by typing should trigger onValueChange (T833415)", function(assert) {
```
See details in #10797.